coordinator/src/bin/coordinator.rs

```diff
@@ -210,6 +210,9 @@ async fn main() -> Result<()> {
 
     let app = router(node, pool, settings);
 
+    // Start the metrics exporter
+    let _exporter = autometrics::global_metrics_exporter();
```

Comment:
:question: Should we use a CLI flag to activate this?

Reply:
had a quick look and it complicates things (ie. what should we do with the endpoint?). I'll enable it for now by default, and let's see whether it has any impact on usage.